### PR TITLE
add missing rel=help spec links

### PR DESCRIPTION
**Pull request by svgeesus**
rel=help documents what is being tested, and automatically decorates the spec with test results

### CODE REVIEW

**Review by drott — APPROVED** [review]

File: css/css-fonts/variations/variable-box-font.html
Comment (jfkthame) [inline css/css-fonts/variations/variable-box-font.html]: Shouldn't this link to the *property* at https://www.w3.org/TR/css-fonts-4/#font-variation-settings-def rather than the *descriptor* section?

File: css/css-fonts/variations/variable-gpos-m2b.html
Comment (jfkthame) [inline css/css-fonts/variations/variable-gpos-m2b.html]: Again, this is testing the *property* and not the *descriptor*.

File: css/css-fonts/variations/variable-gsub.html
Comment (jfkthame) [inline css/css-fonts/variations/variable-gsub.html]: And again.

**Review by jfkthame — COMMENTED** [review]
I think some of these aren't pointing to the most appropriate section of the spec; see comments inline.

**Comment by svgeesus** [comment]
Thanks for the review @jfkthame links fixed now

**Comment by sideshowbarker** [comment]
@jfkthame I think this PR is waiting on re-review from you

**Review by jfkthame — APPROVED** [review]
LGTM, thanks.

**Pull request merged**
